packages/commonwealth/scripts/refresh-all-memberships.ts

```diff
@@ -13,18 +12,29 @@ async function main() {
 
   const banCache = new BanCache(models);
 
-  const communitiesController = new ServerCommunitiesController(
-    models,
-    banCache,
-  );
-
   const groupsController = new ServerGroupsController(models, banCache);
 
-  const communitiesResult = await communitiesController.getCommunities({
-    hasGroups: true,
+  // fetch all communities via pagination
+  const limit = 50;
+  let cursor = 1;
+  const communitiesResult = await query(Community.GetCommunities(), {
```

Comment:
I see the purpose here, but I think that:

1. If we go with this approach, I think we shouldn't paginate, because It will require more round trip requests to the DB.
2. It might be better just to write some sql here to pull all communities with a group. Something like:
```typescript
const communities = models.query(`
SELECT c.community_id FROM "Communities" c
JOIN "Groups" g ON g.community_id = c.id;
`);
```

This way we will also only be querying for the community_id instead of pulling all the unneeded columns.

Reply:
This script should never need to be run again, and if it is run again, it will be run infrequently. I don't think it's worth optimizing.